server/routes/appointments/bulk-appointment-details/handlers/bulkAppointmentMovementSlip.ts

```diff
@@ -9,5 +10,14 @@ export default class BulkAppointmentMovementSlipRoutes {
     )
 
     res.render('pages/appointments/movement-slip/bulk-appointment', { bulkAppointment })
+    const properties = {
+      username: res.locals.user.username,
+      prisonCode: bulkAppointment.prisonCode,
+      bulkAppointmentId: bulkAppointment.id.toString(),
+    }
+    const metrics = {
+      appointmentCount: 1.0,
```

Comment:
Appointment count should be `bulkAppointment.occurrences.length` as in the number of appointments (occurrences in the back end) within the bulk appointment

Reply:
Looking at this again, using metric name `movementSlipCount` is more accurate. Please could you change `appointmentCount` -> `movementSlipCount`

Reply:
Done